src/CMakeLists.txt

```diff
@@ -120,6 +136,9 @@ if(BUILD_CLIENT)
 endif()
 
 
+set(DISPLAY_DENSITY_FACTOR "1" CACHE STRING "Factor To Multiply Display Density By")
```

Comment:
this should absolutely not be a compile time option

Reply:
What should it be then? A config option?

Reply:
Yes, that'd be better.

Reply:
This is now a config option.